binding/ios/README.md

```diff
@@ -0,0 +1,72 @@
+# Leopard Binding for iOS
+
+## Leopard Speech-to-Text Engine
+
+Made in Vancouver, Canada by [Picovoice](https://picovoice.ai)
+
+Leopard is an on-device speech-to-text engine. Leopard is:
+
+- Private; All voice processing runs locally.
+- Accurate [[1]](https://github.com/Picovoice/speech-to-text-benchmark#results)
+- Compact and Computationally-Efficient [[1]](https://github.com/Picovoice/speech-to-text-benchmark#results)
+- Cross-Platform:
+    - Linux (x86_64)
+    - macOS (x86_64, arm64)
+    - Windows (x86_64)
+    - Raspberry Pi (4, 3)
+    - NVIDIA Jetson Nano
+    - Android and iOS
+
+## Installation
+
+The Leopard iOS binding is available via [Cocoapods](https://cocoapods.org/pods/Leopard-iOS). To import it into your iOS project, add the following line to your Podfile: 
+
+```ruby
+pod 'Leopard-iOS'
+```
+
+## AccessKey
+
+The Leopard SDK requires a valid `AccessKey` at initialization. `AccessKey`s act as your credentials when using Leopard SDKs.
+You can create your `AccessKey` for free. Make sure to keep your `AccessKey` secret.
+
+To obtain your `AccessKey`:
+1. Login or Signup for a free account on the [Picovoice Console](https://picovoice.ai/console/).
+2. Once logged in, go to the [`AccessKey` tab](https://console.picovoice.ai/access_key) to create one or use an existing `AccessKey`.
+
+## Usage
+
+Add the Leopard model file to `Xcode` and create an instance of the engine:
```

Comment:
can we write a couple of sentences here about:

1- default is under lib/common
2- how to put it in the bundle (where? ) if it makes sense?
3- link to console to create their own ... obviously keep this vague for now.